### DISCUSSION TITLE
"getActivePinia()" was called but there was no active Pinia. Are you trying to use a store before calling "app.use(pinia)"? See https://pinia.vuejs.org/core-concepts/outside-component-usage.html for help. This will fail in production.

### DISCUSSION THREAD

**Discussion by bbhxwl** [vuejs/pinia#2817] [Help and Questions] [answered]
"getActivePinia()" was called but there was no active Pinia. Are you trying to use a store before calling "app.use(pinia)"? See https://pinia.vuejs.org/core-concepts/outside-component-usage.html for help. This will fail in production.
![image](https://github.com/user-attachments/assets/b341b755-82dd-4bbd-9511-53fd736171ed)

**Comment by posva** [comment, marked as answer]
Nobody will be able to help you if you don't give more information like a reproduction
